tests/src/Behat/Content/Node/EventContentContext.php

```diff
@@ -67,6 +67,7 @@ public function alterEventFields(BeforeParseEntityFieldsScope $scope): void {
       'Teaser' => 'oe_teaser',
       'Venue' => 'oe_event_venue',
       'Contact' => 'oe_event_contact',
+      'Programme' => 'oe_event_programme',
```

Comment:
Looks like also switch part should be updated (should be with 'Venue' and 'Contact' entity types)